### DISCUSSION TITLE
tried everything but not able to resolve this error

### DISCUSSION THREAD

**Discussion by ayushtiwari24** [honojs/hono#3282] [General]
npm run dev

> dev
> wrangler dev src/index.ts

 ⛅️ wrangler 3.70.0 (update available 3.72.0)
-------------------------------------------------------

⎔ Starting local server...
╭─────────────────────────────────────────────────────────────────╮
│ [b open a    [d open      [l turn off loca[c clear     [x to    │
│   browser,     Devtools,     mode,          console,     exit   │
╰─────────────────────────────────────────────────────────────────╯
╭─────────────────────────────────────────────────────────────────╮│ [b open a    [d open      [l turn off loca[c clear     [x to    ││   browser,     Devtools,     mode,          console,     exit   │╰─────────────────────────────────────────────────────────────────╯╰─────────────────────────────────────────────────────────────────╯C:\Users\tiwar\OneDrive\Desktop\medium-blog\backend\node_modules\wrangler\wrangler-dist\cli.js:29765
            throw a;
            ^

MiniflareCoreError [ERR_RUNTIME_FAILURE]: The Workers runtime failed to start. There is likely additional logging output above.       
    at #assembleAndUpdateConfig (C:\Users\tiwar\OneDrive\Desktop\medium-blog\backend\node_modules\miniflare\dist\src\index.js:9178:13)    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Mutex.runWith (C:\Users\tiwar\OneDrive\Desktop\medium-blog\backend\node_modules\miniflare\dist\src\index.js:3521:16) {   
  code: 'ERR_RUNTIME_FAILURE',
  cause: undefined
}

Node.js v20.16.0

**Comment by ghost** [comment]
Hi @ayushtiwari24 

It seems to be the same error as https://github.com/honojs/hono/issues/3269.
Try the solution described in https://github.com/cloudflare/workers-sdk/issues/4709#issuecomment-2289638966.

**Reply by ghost** [reply]
If this does not work for you, please contact the [cloudflare/workers-sdk](https://github.com/cloudflare/workers-sdk) community for help. You will get better support than here.
